src/DbPostgreSQLAdapter/RecordSet.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "DbAdapterInterface/IRecordSet.h"
+
+typedef struct pg_result PGresult;
```

Comment:
Is this typedef needed?

Reply:
I'll check it, it does not seem necessary

Reply:
You were right, this was not necessary. I'll remove it